<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start To Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>851</width>
      <height>681</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget#tabWidget{
	background-color:none;
}
/*设置窗体样式*/
QTabWidget#tabWidget::pane{ 
	/*border:none;*/
   border-image: url(:/image/bg3.jpg) 0 0 0 0     stretch stretch;
}
/*设置QTabWidget 的Tab标签所在那一行的样式下*/
QTabWidget#tabWidget::tab-bar{
	alignment:center;	
}
/*设置QTabWidget 的Tab标签的样式*/
QTabWidget#tabWidget QTabBar::tab{
	border-color: black;
	border-bottom:none;
	border-top-left-radius: 6px;
	border-top-right-radius: 6px;
	background:transparent;
	min-width:5ex;
	min-height:5ex;
}
QTabWidget#tabWidget&gt;QTabBar::tab:!selected:hover{
	border:3px solid #C589C3;
	background:rgb(255, 255, 255, 100);
	border-color: black;
	width:40px;
	height:40px;
}
QTabWidget#tabWidget&gt;QTabBar::tab:selected{
background: none;
border:none;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>1</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>0</y>
        <width>643</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>201</width>
        <height>631</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
	border:1px solid gray; 
	outline:0px;
	/*background: rgb(186, 189, 182);*/
}
QListWidget::Item:hover{
  /* background-color: rgb(136, 138, 133);*/
}
QListWidget::Item:selected {
	/*background-color: rgb(136, 138, 133);*/
}
QListWidget::Item:selected:!active {
   /* background-color:rgb(211, 215, 207);*/
}
</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>2</string>
     </attribute>
     <widget class="QWidget" name="widget_tab_2" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>611</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>581</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTreeWidget
{
   border-style:none;   
}

QTreeWidget::item
{
   height: 26;
   color:rgba(255, 255, 255, 0);    
}

QTreeWidget::item:selected:active
{
    background-color: rgba(135, 206, 250, 153);
}

QTreeWidget::item:selected:!active, QTreeView::item:hover 
{
    background-color: rgba(135, 206, 250, 50);
}</string>
      </property>
      <column>
       <property name="text">
        <string>联系人</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>570</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(0,229,238);
border-image: url(:/image/add.png);
}
QPushButton#pushButton:hover
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(142,229,238);
}
QPushButton#pushButton:pressed
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(85,170,255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="icon">
      <iconset theme=":/image/chat.png">
       <normaloff>.</normaloff>.</iconset>
     </attribute>
     <attribute name="title">
      <string>3</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="Avatar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 20pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
